tests/test_trnsys.py

```diff
@@ -23,6 +32,48 @@ def step_forward(self, steps: int) -> (bool, int):
             return (False, 1)
         return (True, 0)
 
+    def get_output_value(self, unit: int, output_number: int) -> (float, int):
+        if unit not in self.units:
+            return (math.nan, 1)
+        key = f"{unit}:{output_number}"
+        value = self.unit_outputs.get(key)
+        if value is None:
+            return (math.nan, 2)  # output is not available for this unit
+        return (value, 0)
+
+    def set_input_value(self, unit: int, input_number: int, value: float) -> int:
+        if unit not in self.units:
+            return 1
+        key = f"{unit}:{input_number}"
+        if key not in self.unit_inputs:
+            return 2  # treat this is though unit does not have this input number
+        self.unit_inputs[key] = value
+
+    def add_unit_to_deck(self, unit: int):
+        """Make a unit be present in the deck."""
+        self.units.add(unit)
+
+    def set_unit_output_value(self, unit: int, output_number: int, value: float):
```

Comment:
Should this return an int to be consistent with the setter above?

Reply:
No, because it's related to setting the mocked simulation state and is different than the actual getters and setters that reach into TRNSYS.  But maybe there's a better way to do what I'm trying to do with this method that we can discuss next time we pair.